src/server/game/World/World.cpp

```diff
@@ -666,6 +670,16 @@ void World::LoadConfigSettings(bool reload)
         LOG_ERROR("server.loading", "PlayerSave.Stats.MinLevel ({}) must be in range 0..80. Using default, do not save character stats (0).", _int_configs[CONFIG_MIN_LEVEL_STAT_SAVE]);
         _int_configs[CONFIG_MIN_LEVEL_STAT_SAVE] = 0;
     }
+    /*
+    _int_configs[CONFIG_INTERVAL_GRIDCLEAN] = sConfigMgr->GetOption("GridCleanUpDelay", 5 * MINUTE * IN_MILLISECONDS);
+    if (_int_configs[CONFIG_INTERVAL_GRIDCLEAN] < MIN_GRID_DELAY)
+    {
+        LOG_ERROR("server.loading", "GridCleanUpDelay (%i) must be greater %u. Use this minimal value.", _int_configs[CONFIG_INTERVAL_GRIDCLEAN], MIN_GRID_DELAY);
+        _int_configs[CONFIG_INTERVAL_GRIDCLEAN] = MIN_GRID_DELAY;
+    }
+    */
```

Comment:
?

Reply:
Why not implement this here

Reply:
Right, as mentioned under **Known Issues and TODO List** above, I never got to do this because I wasn't sure if the order of the WorldIntConfigs enum matters and also in general if there were anything else to take into consideration when adding new configs.

I will have to look into this and implement it.

Reply:
nope. Just add it somewhere. The order of the enumerations don't matter here

Reply:
Got it. Good to know 👍